cmake/test-files.cmake

```diff
@@ -43,9 +43,6 @@ set(MBGL_TEST_FILES
     test/sprite/sprite_image.test.cpp
     test/sprite/sprite_parser.test.cpp
 
-    # src
-    test/src/main.cpp
```

Comment:
This file is generated, you have to change the script too to make sure it won't be added again.